strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/support/TrustStoreCertificationAdditionException.java

```diff
@@ -0,0 +1,13 @@
+package org.carlspring.strongbox.services.support;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class TrustStoreCertificationAdditionException extends RuntimeException
```

Comment:
I think this name could be something more like `TrustStoreCertificateManagementException`...?